test/i18n_backend_test.rb

```diff
@@ -0,0 +1,30 @@
+require "test_helper"
+
+class Rafinesque::I18nBackendTest < Minitest::Test
+
+  def test_has_default_interpolations
+    setup!
+    I18n.with_locale(:wk) do
+      assert_equal I18n.t("something"), "MyBrand is for specialists"
+    end
+  end
+
+  def test_still_handles_interpolations
+    setup!
+    I18n.with_locale(:wk) do
+      assert_equal I18n.t("something_with_placeholder", name: "John"), "Hello John, MyBrand is for the specialist you dream to be"
+    end
+  end
+
+  private
+
+  def semantics_json
+    @semantics_json ||= {"wk" => {"brand" => "MyBrand", "experts" => "specialists", "the expert" => "the specialist"}}
```

Comment:
Line is too long. [118/80]<br>Space inside { missing.<br>Space inside } missing.